relatio/utils.py

```diff
@@ -302,6 +412,44 @@ def prettify(narrative) -> str:
 
     return pretty_narrative
 
+def prettify_narratives(narratives: List[Dict], fix_grammar: bool = False):
+    """ 
+    
+    'Prettifies' a given list of narratives.
+    
+    Converts each narrative from a dictionary to a (grammatically incorrect)
+    human-readable phrase. The grammar can optionally be corrected. It takes
+    roughly one minute to correct the grammars of around 250 narratives.
+
+    Args:
+        narratives (List[Dict]): list of narratives.
+        fix_grammar (bool): if True, the prettified narratives will be 
+            grammatically correct. False by default.
+
+    Returns:
+        pretty_narratives (Counter[str]): a counter over the prettified list of narratives.
+    
+    """
+
+    prettifiable_narratives = []
+    for narrative in narratives:
+        if narrative.get('ARG0') is not None:
+            if narrative.get('B-V') is not None:
+                if narrative.get('ARG1') is not None:
+                    prettifiable_narratives.append(narrative)
+    
+    if fix_grammar:
```

Comment:
See the comment above. Let's fix `character_matching()` and remove this line of code.